from app.seedwork.domain.exceptions import DomainException


class StoryValidationError(DomainException):
    pass
